lib-ee/emqx_ee_connector/src/emqx_ee_connector_tdengine.erl

```diff
@@ -49,6 +51,8 @@ add_default_username(Fields) ->
         fun
             ({username, OrigUsernameFn}) ->
                 {username, add_default_fn(OrigUsernameFn, <<"root">>)};
+            ({password, OrigUsernameFn}) ->
+                {password, add_default_fn(OrigUsernameFn, ?DEFAULT_PASSWORD)};
```

Comment:
Concept of _default password_ sounds somewhat strange to me, honestly. I think that if there's a passwordless auth, then connector should anticipate password being not defined. If there is no passwordless auth, then we should just mark the field required?

Reply:
Usually, The Tdengine use a default password, "data", users don't need to care about this password,  and the schema of `password` is not required, of course, we can change it to true, but in this way, users have to repeat the password which should have omitted.
So we should not force users to input this password again.

Reply:
I see, but I wouldn't really expect users to deploy TDengine DB with a default password in the wild.

Reply:
https://github.com/emqx/emqx/pull/10269